rust-src/concordium_base/src/constants.rs

```diff
@@ -31,8 +29,5 @@ pub const MAX_URL_TEXT_LENGTH: usize = 2048;
 /// Size of the sha256 digest in bytes.
 pub const SHA256: usize = 32;
 
-/// The highest amount of energy allowed when invoking a smart contract endpoint
-/// with a concordium node.
-pub const MAX_ALLOWED_INVOKE_ENERGY: Energy = Energy {
```

Comment:
Why remove this constant? Was this moved somewhere else?

Reply:
It is no longer relevant. The way the API on the node now does not make use of this.